lib/BuildSystem/BuildSystem.cpp

```diff
@@ -2194,7 +2194,7 @@ class SwiftGetVersionCommand : public Command {
     // FIXME: Need a decent subprocess interface.
     SmallString<256> command;
     llvm::raw_svector_ostream commandOS(command);
-    commandOS << executable;
+    commandOS << basic::shellEscaped(executable);
     commandOS << " " << "--version";
 
     // Read the result.
```

Comment:
popen() should ideally be fixed to take an argument vector, but this seems fine for now.